libs/net/dal/Services/UserService.cs

```diff
@@ -40,14 +40,19 @@ public IPaged<User> Find(UserFilter filter)
             query = query.Where(c => EF.Functions.Like(c.FirstName.ToLower(), $"{filter.FirstName.ToLower()}%"));
         if (!String.IsNullOrWhiteSpace(filter.LastName))
             query = query.Where(c => EF.Functions.Like(c.LastName.ToLower(), $"{filter.LastName.ToLower()}%"));
-
+        if (!String.IsNullOrWhiteSpace(filter.Keyword))
```

Comment:
You're converting keyword to lower multiple times.  Do it once.

Reply:
If you have `filter.Keywork.ToLower()` multiple times you are reducing performance.  You should only do this once.